### PR TITLE
Fix ftw.addressblock

### PR DESCRIPTION
**Pull request by busykoala**
Close https://github.com/4teamwork/izug.organisation/issues/1622

### CODE REVIEW

File: docs/HISTORY.txt [hunk 1]
```diff
@@ -2,13 +2,16 @@ Changelog
...
+2.0.0 (unreleased)
 ------------------
 
 - Check for ftw.subsite installed before using its interface. [busykoala]
 - Remove obsolete addressblock.js - also from legacy bundle, since this lead
-  to the fact, that the mapblock.js was registered twice, which lead to the fact, 
+  to the fact, that the mapblock.js was registered twice, which lead to the fact,
   that the whole map implementation was broken. [mathias.leimgruber]
+- Fix: make ftw.addressblock.geo plone5 compatible.
+  There is still an issue with the initial overlay -> `Issue 31<https://github.com/4teamwork/ftw.addressblock/issues/31>`_.
```
Comment (maethu) [inline docs/HISTORY.txt]: This is fixed and can be removed 😄

File: ftw/addressblock/geo/hooks.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        clean_plone4_registry(site)
+
+
+def clean_plone5_registry(site):
+    pass
+
+
+def clean_plone4_registry(site):
+    pass
+    #registry = getUtility(IRegistry)
```
Comment (maethu) [inline ftw/addressblock/geo/hooks.py]: remove ✂️

File: ftw/addressblock/geo/profiles/default_plone5/registry.xml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<registry>
+
+    <records interface="collective.geo.settings.interfaces.IGeoFeatureStyle">
+        <value key="map_viewlet_position">plone.belowcontentbody</value>
+    </records>
+
+    <records prefix="plone.resources/addressblock-geo-js"
+             interface="Products.CMFPlone.interfaces.IResourceRegistry">
+        <value key="js">++resource++ftw.addressblock.geo/addressblock.js</value>
```
Comment (maethu) [inline ftw/addressblock/geo/profiles/default_plone5/registry.xml]: I removed the addressblock.js with my PR https://github.com/4teamwork/ftw.addressblock/pull/32/files

Please don't re-add it again ;-)

File: ftw/addressblock/geo/profiles/uninstall/jsregistry.xml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<object name="portal_javascripts">
+
+  <javascript
+      id="++resource++ftw.addressblock.geo/addressblock.js"
```
Comment (maethu) [inline ftw/addressblock/geo/profiles/uninstall/jsregistry.xml]: See https://github.com/4teamwork/ftw.addressblock/pull/32/files it's already gone

**Comment by busykoala** [comment]
@maethu, I tried to react to your requests in 2f2626d

**Pull request merged**